app/components-react/windows/settings/Output.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { IObsFormType, ObsGenericSettingsForm } from './ObsSettings';
+import { useObsSettings } from './useObsSettings';
+
+export function OutputSettings() {
+  const { settingsFormData } = useObsSettings();
+
+  const type = settingsFormData[0].parameters[0].currentValue === 'Simple' ? 'collapsible' : 'tabs';
+
+  return <ObsGenericSettingsForm type={type as IObsFormType} />;
```

Comment:
looking at how light this component is i wonder if it would make more sense to do more of the output settings specific layout handling here with the tabs and collapsible instead of in `ObsGenericSettingsForm`, it feels a bit weird to me to put a lot of things specific to one settings page in a generic component

Reply:
Using this generic settings form and passing in the backend settings all at once means that the forms will automatically update on the frontend. This will be less frontend maintenance going forward when adding/updating features with settings because all we will need to do is update the realm and the fields will adjust accordingly. Moreover, adding form types to the generic settings form enables the flexibility with form layout without requiring a lot of work on the frontend since all that is needed is to pass in the type. That being said, I will refactor the `ObsTabbedFormGroup` to be less output settings specific.